studio/pages/auth/policies.js

```diff
@@ -0,0 +1,61 @@
+import AuthLayout from '../../components/layouts/AuthLayout'
+import Loading from '../../components/utils/Loading'
+import Error from '../../components/utils/Error'
+import { fetchOpenApiSpec } from '../../lib/api'
+import { Card, Button, Badge, IconSearch, Input } from '@supabase/ui'
+
+export default function Home() {
+  const { tables, isLoading, error } = fetchOpenApiSpec()
```

Comment:
Ah - for this we don't actually use the OpenApiSpec for fetching tables. WE will use PostgresMeta (https://github.com/supabase/postgres-meta/)

This is because the OpenAPI is subject to auth changes, and we need to bypass all auth so that we can see the tables. 

Perhaps you can just pull this out and leave the headings?

Reply:
Done